developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -129,132 +101,308 @@ Once it has been instantiated, you will notice that the client API is different
 ## Major changes
 
 From a user's perspective, major changes with the `v3` client include:
- 
 
 ### Better Typescript Support
 
-  - generics, functions and methods
-
-The `v3` client introduces an extensive set of helper classes to interact with Weaviate. These classes are used to provide strong typing, and to make the client more user-friendly such as through IDE autocompletion.
+The `v3` client introduces better Typescript support. A lot of these benefits are visible with better IDE intellisense and type-safety. To add to this, we've made it possible for users to define their own Generics.
```

Comment:
Consider:

The `v3` client has better Typescript support. The user-facing benefits include better IDE intellisense, improved type-safety, and user-defined generics.